langkit/dsl.py

```diff
@@ -958,6 +959,15 @@ def __new__(mcs, name, bases, dct):
                 )
                 value._name = names.Name.from_lower(key)
                 values.append(value)
+
+                # If this is the default value for this enum type, store it
+                if value.is_default_val:
+                    check_source_language(
+                        default_val_name is None,
+                        'Only one default value is allowed'
+                    )
```

Comment:
Can you add a testcase that show that this check works when there are more than one default value? `properties/invalid_enum_dflt` for instance, just calling emit_and_print_errors, like other such validation tests.

Reply:
I'll add it if you want, however I'm not sure of the value at this stage:

1. This check is going to bounce when we transition to the concrete syntax, probably replaced by a check in lkt.
2. The test I add will need to be rewritten manually, because we cannot unparse those, because those checks abort before we can even emit the .lkt file

Reply:
I think the value of this one is the same as for the other validation checks: make sure that the feature works as expected now, and help to cover this case in the future (especially when migrating to Lkt). This test is supposed to be very easy to write, and so porting this one (and all other such tests) manually should be easy enough.

This is similar to the invalid grammar tests we had to transition to Lkt: it took 1.5 days to port them all, but having them has value to validate the Lkt implementation (and did help us find bugs and missing features!).

Reply:
Ok, will go ahead then

Reply:
Thanks!